<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PedidosPendientesForm</class>
 <widget class="QWidget" name="PedidosPendientesForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tablaPedidos">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>441</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>119</width>
     <height>57</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QComboBox" name="productosComboBox"/>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="unidadesSpinBox"/>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>AÃ±adir Pedido</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
